app/controllers/concerns/project_concern.rb

```diff
@@ -0,0 +1,13 @@
+module ProjectConcern
+  extend ActiveSupport::Concern
+
+  private
+
+  def change_date(project)
```

Comment:
@linesarefuzzy suggested using an inheritance pattern instead of a concern for certain methods like this. However, @hooverlunch, we appear to be going the way of concerns based on a previous code review.